monix-stream/src/test/scala/neotypes/monix/stream/MonixObservablesSuite.scala

```diff
@@ -17,6 +17,8 @@ object MonixObservablesTestkit extends StreamTestkit[MonixStream, Task](MonixTas
 
       override final val streamInstance: Stream.Aux[MonixStream, Task] =
         implicitly
+
+      override def streamConcurrently(stream1: MonixStream[Unit], stream2: MonixStream[Unit]): MonixStream[Unit] = stream1.combineLatest(stream2).map(_ => ())
```

Comment:
Looking at the [documentation of **Monix**](https://monix.io/api/current/monix/reactive/Observable.html) it seems that while this may work for our case it is not really the same operation we wanted.
I think it would be better to use:

```scala
override def streamConcurrently(stream1: MonixStream[Unit], stream2: MonixStream[Unit]): MonixStream[Unit] =
  Observable(stream1, stream2).merge
```